src/index.html

```diff
@@ -11,7 +11,194 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="block">
+        <legend>
+          Personal information
+        </legend>
+        <label for="surnameId" class="input">
+          Surname:
+          <input
+            type="text"
+            id="surnameId"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label for="nameId" class="input">
+          Name:
+          <input
+            type="text"
+            id="nameId"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label for="oldId" class="input">
+          How old are You?
+          <input
+            type="number"
+            id="oldId"
+            name="old"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label for="birthDateId" class="input">
+          Full date of birth:
+          <input
+            type="date"
+            id="birthDateId"
+            name="birthDate"
+            min="1900-01-01"
+            max="2021-10-12"
+          >
+        </label>
+        <label for="acceptId" class="input">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="acceptId"
+            name="accept"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend>
+          Registration
+        </legend>
+        <label for="emailId" class="input">
+          E-mail:
+          <input
+            type="email"
+            id="emailId"
+            name="emailRegistration"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label for="passwordId" class="input">
+          Password:
+          <input
+            type="password"
+            id="passwordId"
+            name="passwordRegistration"
+            minlength="8"
+            maxlength="12"
+            autocomplete="off"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="block">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <label for="loveCatsYes">
+          Do you love cats?
+        </label>
+        <label for="loveCatsYes">
+          <input
+            type="radio"
+            id="loveCatsYes"
+            name="loveCats"
+          >
+            Yes
+        </label>
```

Comment:
It'd be better to put text first, also fix your indents
```suggestion
        <label for="loveCatsYes">
        	Yes
          <input
            type="radio"
            id="loveCatsYes"
            name="loveCats"
          >
        </label>
```